crates/bevy_reflect/src/reflect.rs

```diff
@@ -313,6 +313,24 @@ where
         })
     }
 
+    /// For a type implementing [`PartialReflect`], combines `reflect_clone` and
```

Comment:
Nit: Is this meant to be read as ``For a type,`T`, implementing...`` or ``For a type,`self`, implementing...``? We should probably clarify here.

If the former, we also have to be mindful of explaining why `T` should generally be `PartialReflect` since it's not included as a type bound (which it rightfully shouldn't to allow for remote types).

If the latter, it might be better to leave out the qualification since even as a dyn-incompatible method, `self` will still need to be `PartialReflect`.

Reply:
Again, the comment is confusing because it's out of date relative to the code. I will fix.